orienteer-core/src/main/java/org/orienteer/core/component/property/UnregistredPropertyEditPanel.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * 
+ */
+package org.orienteer.core.component.property;
+
+import org.apache.wicket.Component;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.ajax.markup.html.AjaxLink;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.link.Link;
+import org.apache.wicket.markup.html.panel.GenericPanel;
+import org.apache.wicket.model.IModel;
+import org.orienteer.core.behavior.UpdateOnActionPerformedEventBehavior;
+
+import com.orientechnologies.orient.core.db.document.ODatabaseDocument;
+import com.sun.mail.handlers.image_gif;
+
+import ru.ydn.wicket.wicketorientdb.OrientDbWebSession;
+import ru.ydn.wicket.wicketorientdb.proto.IPrototype;
+
+/**
+ * @author Asm
+ *
+ */
+public class UnregistredPropertyEditPanel<T> extends GenericPanel<T> {
+
+	IModel<T> propertyModel;
+	IModel<?> reloadableModel;
+	Component successMessage;
+	Component reloadableComponent;
+	
+	public UnregistredPropertyEditPanel(String id, IModel<T> model, Component propertyComponent) {
+		super(id, model);
+		//add(new Label("propertyComponent","cool property component"));
+		//add(propertyComponent);
+		propertyModel = model;
+		propertyComponent.setMarkupId(id+":propertyComponent");
+	}
+
+	public UnregistredPropertyEditPanel(String id, IModel<T> model) {
+		super(id, model);
+		propertyModel = model;
```

Comment:
You have already populeted model to super()... It means that it's meaningless to have that doubled in "propertyModel". You can just use getModel(), getModelObject() and etc.